CONTRIBUTING.md

```diff
@@ -37,13 +49,13 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 
    > If you are using Yarn v2, run `yarn set version classic` to change to Yarn Classic.
 
-   > Ensure that [Yarn global bin](https://classic.yarnpkg.com/en/docs/cli/global) is added to your PATH. For example, add `export PATH="$(yarn global bin):$PATH"` to your shell profile file on Linux or macOS.
+   > Ensure that `.bin` directory is added to your PATH. For example, add `export PATH="<amplify-cli/.bin>:$PATH"` to your shell profile file on Linux or macOS.
```

Comment:
Should this be added the one of the setup scripts?

Reply:
we did not used to do it before, and in my opinion the setup script should not mess with local env variables, but I don't have strong opinions about this

Reply:
I would add it to the setup scripts to remove an extra step that contributors have to do in order to setup their local environment